docs/preview/abac-repo-permissions/README.md

```diff
@@ -358,6 +358,38 @@ By following these steps, users will have read access to all repositories with t
 
 ![Create the following ABAC rule with a negation rule to exclude the repository application/secret when granting read permissions.](./media/14-abac-condition-to-exclude-specific-repository-application-secret.png)
 
+#### Scenario: Different ABAC conditions for different actions
```

Comment:
Let's change the title to `#### Scenario: Setting ABAC Conditions for a Role with Multiple Actions`